internal/util/reports.go

```diff
@@ -10,8 +11,10 @@ import (
 
 var (
 	reportsHost  = "kong-hf.konghq.com"
-	reportsPort  = 61829
+	reportsPort  = 61833
 	pingInterval = 3600
+	tlsConf      = tls.Config{MinVersion: tls.VersionTLS12}
+	dialer       = net.Dialer{Timeout: time.Second * 30}
```

Comment:
non-blocker: Before by nature of UDP we didn't really have any concerns about a timeout, but now it seems we do. Not a blocker, and I assume that the number we have here stems from testing we've done, but I just want to ensure that we're being deliberate about this setting as there are some possible implications.

Reply:
The timeout choice was "arbitrary not infinite value", and 30s felt longer than I'd reasonably expect a TCP connection to complete its handshake. Are there some other criteria we should consider?

Reply:
The average server response times should be considered, and then it's _slightly_ stretching the scope of this PR but solid retry management and logging logic for this code might improve loose ends.

Reply:
Possibly the average response time of the server, but I have every reason to believe 30s will cover that, especially with retries. nm